V3 (#19)
Financial-Times/o-loading: MIGRATION.md

## Migration Guide

### Migrating from 2.X.X to 3.X.X

V3 changes the internal structure of `o-loading`.

These mixins have been privatised and are no longer meant to be used within a product.
```diff
- oLoadingColor
+ _oLoadingColor
- oLoadingSize
+ _oLoadingSize
```

In order to output styles, the available mixin is now:
```scss
@include oLoading($opts: (
	'colors': ('dark'),
	'sizes': ('small')
))
```

> In order to add styles to a specific element, you can use:

Reviewer: Maybe mention why choose to do it this way?
```suggestion
If you can't use `o-loading` classes, add styles to a specific element with `oLoadingContent`:
```
